packages/flutter_tools/lib/src/globals.dart

```diff
@@ -284,5 +284,5 @@ PreRunValidator get preRunValidator => context.get<PreRunValidator>() ?? const N
 const String kDefaultFrameworkChannel = 'master';
 
 // Used to build RegExp instances which can detect the VM service message.
-const String kServicePrefixRegExp = '(?:Observatory|The Dart VM service is)';
+const String kServicePrefixRegExp = '(?:VmService|The Dart VM service is)';
```

Comment:
Should this be `VM Service`, which seems more human?

Reply:
Yes, this just got caught up in my find/replace... :-) We actually don't need this anymore since `Observatory` hasn't been in the VM service message for quite awhile now.